src/ProjectOrigin.WalletSystem.Server/Database/Postgres/Scripts/v2/v2-0004.sql

```diff
@@ -0,0 +1,90 @@
+-- Add type certificate_view
+CREATE VIEW certificates_query_model AS
```

Comment:
Why would you rather have a view than have it in code?

Reply:
since they are using in multiple commands in the repository. Also in the future these might become materialized views for performance using (incremental views)[https://wiki.postgresql.org/wiki/Incremental_View_Maintenance]